src/components/views/TheHeader.vue

```diff
@@ -1,10 +1,22 @@
 <script>
 import { defineComponent } from 'vue';
 import TheContainer from '@/components/common/TheContainer.vue';
+import TopNavbar from '@/components/features/Header/TopNavbar.vue';
+import SignSection from '@/components/features/Header/SignSection.vue';
 
 export default defineComponent({
   components: {
     TheContainer,
+    TopNavbar,
+    SignSection,
+  },
+  computed: {
+    publicContent() {
```

Comment:
Tiny thing, but considering it will be a boolean value, maybe let's rename this to something like `isPublicContent`? 🤔